### PR TITLE
Link to chanjo2 coverage report for genes of the HPO panel:

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- Link to chanjo2 coverage report for genes of the HPO panel. Other partial fix for #4000 

### Main branch:

image

<hr>

### This branch:

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test at CG**:
1. Deploy on cg-vm1.
2. Go to [a case with d4 files](https://scout-stage.scilifelab.se/cust000/F0006358-mip7)
3. Click on the new buttons and make sure they create reports

**How to test at elsewhere**:
1. Deploy
2. Go to a case with samples having d4 files
3. Assign an HPO phenotype to the case and then click on "Create HPO Panel"

image

4. Click on the chanjo2 report button and make sure it's populated

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by Jakob37
- [x] tests executed by Jakob37, CR

### CODE REVIEW

File: scout/server/blueprints/cases/templates/cases/gene_panel.html [hunk 1]
```diff
@@ -91,21 +91,6 @@ <h6 class="ms-3 mt-3"><span class="fas fa-clipboard-list"></span>&nbsp;HPO gene
         {% endif %}
         )
         {% if case.dynamic_gene_list_edited %} <span class="text-danger ml-3"> (edited) </span> {% endif %}
-        {% if case.track != 'cancer' and config.SQLALCHEMY_DATABASE_URI and case.dynamic_gene_list|length > 0 %}
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/gene_panel.html:94]: Moved down as buttons

**Comment by northwestwitch** [comment]
@Jakob37 I've added another part. When you have time, thanks!

**Comment by Jakob37** [comment]
> @Jakob37 I've added another part. When you have time, thanks!

Great, I'll test!

**Comment by Jakob37** [comment]
Not sure if an error in my setup or in Scout/Chanjo2.

When coming back to a running Chanjo2 instance after being away for some time, I get a `Server has gone away` error ("Internal error" message in the browser)

```
api_chanjotest |     raise sqlalchemy_exception.with_traceback(exc_info[2]) from e
api_chanjotest |   File "/venv/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 1969, in _exec_single_context
api_chanjotest |     self.dialect.do_execute(
api_chanjotest |   File "/venv/lib/python3.11/site-packages/sqlalchemy/engine/default.py", line 922, in do_execute
api_chanjotest |     cursor.execute(statement, parameters)
api_chanjotest |   File "/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 179, in execute
api_chanjotest |     res = self._query(mogrified_query)
api_chanjotest |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
api_chanjotest |   File "/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 330, in _query
api_chanjotest |     db.query(q)
api_chanjotest |   File "/venv/lib/python3.11/site-packages/MySQLdb/connections.py", line 255, in query
api_chanjotest |     _mysql.connection.query(self, query)
api_chanjotest | sqlalchemy.exc.OperationalError: (MySQLdb.OperationalError) (2006, 'Server has gone away')
```

Clicking the link again shortly after, it works.

**Comment by Jakob37** [comment]
Also works for real cases!

**Review by Jakob37 — APPROVED** [review]
Looks good to me, the button appears as expected.

**Comment by northwestwitch** [comment]
> api_chanjotest | sqlalchemy.exc.OperationalError: (MySQLdb.OperationalError) (2006, 'Server has gone away')

Mmm interesting, t's not related to the code in Scout but it comes from chanjo2, I'll look into it!

**Comment by northwestwitch** [comment]
> The overview does not appear, but that looks like expected as there is no Chanjo1 coverage here.

I'm going to add another button for the overview in Chanjo2 next week

**Comment by northwestwitch** [comment]
> Also works for real cases!

Nice! Thanks for the review! 🚀

**Pull request merged**